src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,40 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CalculatorTest {
+    @Test
+    public void sum_NullOrEmpty() {
+        assertThat(new Calculator(null).sum()).isEqualTo(0);
+        assertThat(new Calculator("").sum()).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(new Calculator(null).sum()).isZero();
```
이런 방식도 있습니다.
그리고 코드가 중복이 되어있습니다. @NullAndEmptyResource 참고해 보세요 :)